scripts/scil_visualize_bundles_mosaic.py

```diff
@@ -10,44 +10,60 @@
 import argparse
 import logging
 import os
+import random
 import shutil
 
-import nibabel as nib
 from fury import actor, window
+import nibabel as nib
+import numpy as np
 
 from PIL import Image
 from PIL import ImageFont
 from PIL import ImageDraw
 
-from scilpy.io.utils import (
-    add_overwrite_arg, assert_inputs_exist, assert_outputs_exist)
+from scilpy.io.image import get_data_as_mask
+from scilpy.io.utils import (add_overwrite_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist)
+from scilpy.utils.filenames import split_name_with_nii
 
 
 def _build_arg_parser():
-    parser = argparse.ArgumentParser(
+    p = argparse.ArgumentParser(
         description=__doc__, formatter_class=argparse.RawTextHelpFormatter)
-    parser.add_argument('anat_reference',
-                        help='Image used as background (e.g. T1, FA, b0).')
-    parser.add_argument('inputs', nargs='+',
-                        help='List of streamline files supported by nibabel.')
-    parser.add_argument('output_name',
-                        help='Name of the output image mosaic '
-                             '(e.g. mosaic.jpg, mosaic.png).')
-    parser.add_argument('--zoom', type=float, default=1.0,
-                        help='Rendering zoom. '
-                             'A value greater than 1 is a zoom-in, '
-                             'a value less than 1 is a zoom-out.')
-    parser.add_argument('--ttf', default=None,
-                        help='Path of the true type font to use for legends.')
-    parser.add_argument('--ttf_size', type=int, default=35,
-                        help='Font size (int) to use for the legends.')
-    parser.add_argument('--opacity_background', type=float, default=0.4,
-                        help='Opacity of background image [0, 1.0]')
-    parser.add_argument('--resolution_of_thumbnails', type=int, default=300,
-                        help='Resolution of thumbnails used in mosaic.')
-
-    add_overwrite_arg(parser)
-    return parser
+    p.add_argument('anat_reference',
+                   help='Image used as background (e.g. T1, FA, b0).')
+    p.add_argument('inputs', nargs='+',
+                   help='List of streamline or files supported by nibabel '
```

Comment:
is the 'or' supposed to be there?
I would suggest
List of streamline files supported by ...